src/components/organisms/admin/NameAdditionForm.tsx

```diff
@@ -0,0 +1,31 @@
+import { FieldErrors } from 'react-hook-form';
+import {
+  FormControl, FormLabel, Input, Text,
+} from '@chakra-ui/react';
+import { CategoryFormInputData, ProductFormInputData } from '@/types';
+
+interface NameAdditionFormProps {
+  additionType: 'product' | 'category';
+  register: any;
+  errors: FieldErrors<ProductFormInputData | CategoryFormInputData>;
+}
```

Comment:
[공식문서](https://react-hook-form.com/ts#UseFormReturn) 찾아보니 
아래와 같이 작성하는 것 같은데, 혹시 해보셨을까요??

```suggestion
interface NameAdditionFormProps {
  additionType: 'product' | 'category';
  register: UseFormRegister<ProductFormInputData | CategoryFormInputData>;
  errors: FieldErrors<ProductFormInputData | CategoryFormInputData>;
}
```